docs/infrastructure/fern/GENERATING.md

```diff
@@ -0,0 +1,19 @@
+# Generating
+
+## Step 1: Cloud & build Fern
+
+```sh
+gh repo clone rivet-gg/fern
```

Comment:
Can we instead use `git` here? I think people will be less familiar with `gh`, and potentially not know what the command would do.

```suggestion
git clone https://github.com/rivet-gg/fern
```